WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndValueRow.swift

```diff
@@ -69,6 +69,7 @@ struct TitleAndValueRow: View {
 
                     if let titleSuffixImage {
                         titleSuffixImage
+                            .foregroundColor(Color(.brand))
```

Comment:
Just a note: Since we're using this component across multiple features, changing the foreground of the image here might result on unexpected changes somewhere else. In this case I think is fine since (from what I checked) we don't seem to use that `titleSufixImage` property anywhere else except than in the drawer.

Something I noticed in dark mode is that the colour might not be `.brand`, since there's a difference in the purple used for other icons and the one used in the lock:

<img width=600 src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/084374c4-eb61-46df-825e-27603545f45e"/>

This is entirely up to you, but this made me wonder if we could make the component more reusable by changing the property to something like the following, since is already optional:
``` diff
- private let titleSuffixImage: Image?
+ private let titleSuffixImage: (image: Image, withColor: Color)?
```
So we could pass both image and necessary colour to the view element as needed.

Reply:
> `.brand`

What do you think about using `Color(.primary)`?

Reply:
| Light      | Dark |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-01-19 at 11 41 36](https://github.com/woocommerce/woocommerce-ios/assets/6242034/3422c483-e08f-4f0a-ab70-bfa731b35090)      | ![Simulator Screenshot - iPhone 15 Pro - 2024-01-19 at 11 41 42](https://github.com/woocommerce/woocommerce-ios/assets/6242034/0eb214e4-f984-4454-87f0-aeb0b053b446)       |

Reply:
> What do you think about using Color(.primary)?

Yeah, seems like the same one! 👍